core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -205,8 +222,7 @@ private[spark] object ClosureCleaner extends Logging {
 
     // A list of enclosing objects and their respective classes, from innermost to outermost
     // An outer object at a given index is of type outer class at the same index
-    val outerClasses = getOuterClasses(func)
-    val outerObjects = getOuterObjects(func)
```

Comment:
Indeed. I don't remove them now just because there are too many unit test for them. I will take time to remove them and modify these tests. Thanks.